.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: generic
+services:
+  - docker
+
+cache:
+  directories:
+    - $HOME/.ccache
+
+git:
+  quiet: true
+
+env:
+  global:
+    - CATKIN_LINT=true
```

Comment:
Not sure we want to enable this immediately.

@cardboardcode: will the packages as they are pass a `catkin_lint` check?

Reply:
@gavanderhoorn Thank you for the quick response.

The packages will **fail** with `catkin_lint` check.

**Reason**:  There are dependencies on external packages beyond **rvip**. eg.[**darknet_ros**](https://github.com/leggedrobotics/darknet_ros)

**Follow-up**: Will remove those dependencies shortly and update again in this thread.

Reply:
> The packages will **fail** with `catkin_lint` check.

Are you sure? Afaik `catkin_lint` does not check for presence of dependencies. It just checks `package.xml` and `CMakeLists.txt` for problems.

> **Follow-up**: Will remove those dependencies shortly and update again in this thread.

If those dependencies would need to be removed for CI to pass, we need to fix CI (within reason of course), not the dependencies.

Reply:
HI @gavanderhoorn,

The codebase should now pass the `catkin_lint` check with [**the recent commit**](https://github.com/ros-industrial/rvip/pull/1/commits/2e747a5f3b0c4059d6dafcc76f5fc073e2fe699d).

Please proceed with enabling.

Reply:
If you could review this PR (ie: *Accept* it), I can merge the PR.